packages/fxa-content-server/app/tests/spec/lib/router.js

```diff
@@ -514,16 +514,71 @@ describe('lib/router', () => {
   });
 
   describe('React-related methods', () => {
+    beforeEach(() => {
+      const mockAlwaysShownReactGroups = ['simpleRoutes'];
```

Comment:
While this value is currently equal to what we've got in `router.js`, I've mocked it to return this value so that when we add to that array in the future, we won't have to go into our tests and update our route references to something else.